library/include/hipblaslt.h

```diff
@@ -97,6 +97,12 @@ typedef enum {
   HIPBLASLT_MATRIX_LAYOUT_STRIDED_BATCH_OFFSET = 1 /**<Stride (in elements) to the next matrix for the strided batch operation. Default value is 0. Data Type: int64_t*/
 } hipblasLtMatrixLayoutAttribute_t;
 
+/** Pointer mode to use for alpha */
+typedef enum {
+    HIPBLASLT_POINTER_MODE_HOST = 0,                          /** targets host memory */
+    HIPBLASLT_POINTER_MODE_ALPHA_DEVICE_VECTOR_BETA_HOST = 1, /** alpha pointer targets an array in device memory */
```

Comment:
alpha pointer targets a device memory vector of length equal to the number of rows of matrix D, and beta is a single value in host memory.